.github/workflows/release.yml

```diff
@@ -27,7 +27,7 @@ jobs:
           go-version: 1.13
 
       - name: Install upx
-        run: sudo apt update && sudo apt install upx -y
+        run: sudo apt update && sudo apt install libucl1 && curl -L http://archive.ubuntu.com/ubuntu/pool/universe/u/upx-ucl/upx-ucl_3.96-2_amd64.deb -o upx.deb && dpkg -i upx.deb
```

Comment:
sudo for the last cmd? `sudo dpkg -i upx.deb`

Reply:
yes, will push a fix

Reply:
fixed in 209cf27